posts/call-for-maintainers-2020.md

```diff
@@ -0,0 +1,17 @@
+**We're looking for new Track Maintainers. Interested? Read on!**
+
+Exercism is powered by the energy of many volunteers, especially the [track maintainers](https://exercism.io/team/maintainers) who are responsible for creating and managing our Language Tracks.
+
+We're about to start a big push towards version 3 (v3) of Exercism. Getting all the tracks ready for v3 is going to be a big community effort, and we're looking to add more maintainers to oversee and manage that task.
+
+In the push to v3, our maintainers will be responsible for:
+- **Designing the track:** Working out what someone needs to learn to become fluent in a language, and mapping this out as a pathway.
+- **Decision making:** Making design decisions around the tooling the track uses, the standards it conforms to, the defaults it uses, etc.
+- **Encouraging and supporting contributions:** Providing a welcoming environment for people that want to help out, answering their questions, etc.
+- **Reviewing PRs:** A lot of the actual work of building out v3 tracks will be done by the community. Maintainers act as gatekeepers, reviewing the work that the community does and ensuring the track's standards are enforced.
+
+This might all sound a bit daunting but our more experienced team will help you get started and hold your hand as you learn the ropes. Ideally you can commit 3hrs per week (preferably spread over a week) for about 6 months, although obviously life ebs and flows, and some weeks might work better than others. You'll also preferably have some experience of Exercism - either as a mentor or having contributed to the codebase. This isn't strictly a requirement, but it's nice to know you've cared about Exercism for a while.
```

Comment:
> This might all sound a bit daunting

Not everyone has to do all the things. Ideally if we have a solid team of maintainers for a track, then some people will be more excited about some aspects of the work than others.

> Ideally you can commit 3hrs per week

s/3hrs/3 hours/ ?

> You'll also preferably have some experience of Exercism

I think that we should require people to have at the very least _used_ Exercism. This might seem obvious, but I've had people with absolutely no experience with Exercism (they basically didn't even know what it was) reach out to me and ask to be maintainers, because apparently being a maintainer on an open source project carries with it some sort of credibility or status in some circles. Having _contributed_ to Exercism either as a mentor or on GitHub is a plus, but I agree that it's not a requirement.

Reply:
```suggestion
This might all sound a bit daunting but with the work spread between a team of maintainers, individuals can do the bits that they feel best equipped for. Our more experienced maintainers will also help you get started and hold your hand as you learn the ropes. 

To be able to do the role you will:
- Be able to can commit an average of 3 hours per week (preferably spread over the week) for about 6 months. We know that life ebs and flows though so fully understand that some weeks might work better than others for you to contribute. 
- Have used Exercism.
- Preferably have contributed to Exercism - either as a mentor or having contributed to the codebase. While this isn't an absolute requirement, if too many people want to maintain a track, we will give priority to people who have shown a desire to help over the longer term.
```

@kytrinyx Thoughts on that ^^

Reply:
I like the new "contributed to Exercism preference" bit!